### PR TITLE
F#33 componentcard

### LINKED ISSUE #33: As a programmer, I want movie cards to be easily reusable components to reduce the amount of code I need to write
Make cards into web components

### PR DESCRIPTION
**Pull request by samsaksfithian**
closes #33

### CODE REVIEW

File: MovieCard.js [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+		this.render();
+	}
+	
+	// =============================================================
+	
+	render() {
+		this.innerHTML = /* html */ `
+		<div class="grid-item" id="trailer">
+			<iframe
+				width="560"
```
Comment (hanshank) [inline MovieCard.js]: What about changing this to 100%? The embedded youtube player doesn't fill it's container width on my screen.

File: main.js [hunk 2]
```diff
@@ -0,0 +1,19 @@
+// =============================================================
```
Comment (hanshank) [inline main.js]: What are these for? I guess it's in order to separate some code, but I feel like they are redundant.

File: MovieCard.js (hunk 1 above)
Comment (samsaksfithian) [reply MovieCard.js]: Good suggestion, changed!

File: main.js (hunk 2 above)
Comment (samsaksfithian) [reply main.js]: Yeah, it's stylistic to keep everything neat and spaced out and easily readable

**Pull request merged**